### PR TITLE
Update NPD config for GCI

### PR DESCRIPTION
**Pull request by dashpole**
**What this PR does / why we need it**:
Use https://github.com/kubernetes/node-problem-detector/pull/180 on GCI

**Special notes for your reviewer**:
This is currently pending an NPD release.

**Release note**:
```release-note
On COS, NPD creates a node condition for frequent occurrences of unregister_netdevice
```
/assign @Random-Liu 
/sig node
/kind feature
/priority important-soon

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -1181,10 +1181,12 @@ function start-node-problem-detector {
   local -r km_config="${KUBE_HOME}/node-problem-detector/config/kernel-monitor.json"
   # TODO(random-liu): Handle this for alternative container runtime.
   local -r dm_config="${KUBE_HOME}/node-problem-detector/config/docker-monitor.json"
+  local -r custom_km_config="${KUBE_HOME}/node-problem-detector/config/kernel-monitor-counter.json"
   echo "Using node problem detector binary at ${npd_bin}"
   local flags="${NPD_TEST_LOG_LEVEL:-"--v=2"} ${NPD_TEST_ARGS:-}"
   flags+=" --logtostderr"
   flags+=" --system-log-monitors=${km_config},${dm_config}"
+  flags+=" --custom-plugin-monitors=${custom_km_config}"
```
Comment (Random-Liu) [inline cluster/gce/gci/configure-helper.sh]: Let's still add the EXTRA ARG for extensibility.

Comment (dashpole) [reply cluster/gce/gci/configure-helper.sh]: done

**Comment by dashpole** [comment]
/assign @dchen1107 
for approval

**Comment by dashpole** [comment]
/unassign @dchen1107 
/assign @vishh 
since dawn is OOO

File: cluster/gce/gci/configure-helper.sh [hunk 2]
```diff
@@ -1181,13 +1181,18 @@ function start-node-problem-detector {
   local -r km_config="${KUBE_HOME}/node-problem-detector/config/kernel-monitor.json"
   # TODO(random-liu): Handle this for alternative container runtime.
   local -r dm_config="${KUBE_HOME}/node-problem-detector/config/docker-monitor.json"
+  local -r custom_km_config="${KUBE_HOME}/node-problem-detector/config/kernel-monitor-counter.json"
```
Comment (vishh) [inline cluster/gce/gci/configure-helper.sh]: How is this file included in the release? Doesn't it need change to Bazel rules?

Comment (dashpole) [reply cluster/gce/gci/configure-helper.sh]: It is part of the NPD release tarball.

Comment (dashpole) [reply cluster/gce/gci/configure-helper.sh]: So it is added by the change in the NPD version.

**Comment by geekgonecrazy** [comment]
@dashpole any idea when this change actually makes it into gke?  I see 1.10.5-gke.2 just released but its still using node-problem-detector `0.4.1`

It'd be nice to at the very least get this identifying nodes with kernel issues like #64743 and resolve them.

**Comment by dashpole** [comment]
@geekgonecrazy  1.9.10, 1.10.6, 1.11.1, 1.12.0

**Comment by geekgonecrazy** [comment]
@dashpole perfect! Thanks for your response!

**Pull request merged**